### ISSUE TITLE
Sync non existing localSubPath makes an error

### ISSUE THREAD

**Issue by igoooor** [devspace-cloud/devspace#1031]
**What happened?**  
With the following sync configuration:
```
dev:
  sync:
  - imageName: backend
    containerPath: /app/vendor/
    localSubPath: vendor/
    ...
```
and if the local folder `./vendor/` does not exist when starting the sync, the following error happens and the sync stops:
```
[fatal]  start sync: Unable to start sync: start sync: create sync: eval symlinks: lstat vendor: no such file or directory
```

**What did you expect to happen instead?**  
The sync to work, and the container folder to be synced back to the local machine.

**Local Environment:**  
- Operating System: mac
- Deployment method: helm
- devspace version 4.8.0

**Kubernetes Cluster:**  
- Cloud Provider: local with docker for mac
- Kubernetes Version: 1.16

/kind bug

**Comment by FabianKramm** [comment]
@igoooor thanks for reporting! Yes currently the local sub path is not created when it does not exist, however I think we could change this, shouldn't be a problem.

**Comment by FabianKramm** [comment]
Implemented in v4.9.0
